.github/workflows/codespell.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [main]
+  pull_request:
```

Comment:
Our main branch is still called 'master' - does github actions know to translate this?

Reply:
Ah, I missed this! No, this should be changed to master.

Reply:
Fixed.